integration-tests/model/test_nonsplit_physics.py

```diff
@@ -0,0 +1,229 @@
+"""
+This script tests the non-split timestepper against the split timestepper
+using a coupled wave equation, where one of the spatial derivatives
+is a physics parametrisation.
+One split method is tested, whilst different nonsplit IMEX and explicit time
+discretisations are used for the dynamics and physics.
+"""
+
+from firedrake import (SpatialCoordinate, PeriodicIntervalMesh, split,
+                       norm, Constant, cos, sin, Function, Projector, inner, dx)
+from gusto import *
+import pytest
+from math import pi
+
+
+class WaveEquation(PrognosticEquationSet):

```

Comment:
Might we actually want this to be an actual equation rather just live in the tests? I'm happy either way but just wanted to suggest it